scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -785,7 +785,7 @@
                                                     (atom state-locked?)
                                                     dynamic-cluster-config?
                                                     compute-cluster-launch-rate-limiter cook-pool-taint-name cook-pool-taint-prefix cook-pool-label-name
-                                                    (with-meta (repeatedly lock-shard-count #(ReentrantLock.))
+                                                    (with-meta (vec (repeatedly lock-shard-count #(ReentrantLock.)))
```

Comment:
I think a comment explaining why the `vec` is important would be good.

Reply:
Comment added.